src/cli/commands/mod.rs

```diff
@@ -96,13 +96,7 @@ pub async fn cli() {
         Commands::Checkpoint(args) => args.handle(global).await,
     };
 
-    if let Err(e) = r {
-        log::error!(
-            "process command: {:?} failed due to error: {:?}",
-            args.command,
-            e
-        )
-    }
+    r.with_context(|| format!("error processing command {:?}", args.command))
```

Comment:
This is really helpful, thanks!

Reply:
Like I said the other change I would make (one that would probably break the scripts) is to [log to stderr](https://github.com/consensus-shipyard/fendermint/blob/master/fendermint/app/src/main.rs#L22) (which the agent does at the moment already) and [use simple println](https://github.com/consensus-shipyard/fendermint/blob/master/fendermint/app/src/cmd/rpc.rs#L221) to output results. That would make it easier to automate.